CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# How to contribute
+
+cert-manager is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers and other resources to make getting your contribution into cert-manager easier.
+
+# Email and chat
+
+- Email: [giantswarm](https://groups.google.com/forum/#!forum/giantswarm)
+- IRC: #[giantswarm](irc://irc.freenode.org:6667/#giantswarm) IRC channel on freenode.org
+
+## Getting started
+
+- Fork the repository on GitHub
+- Read the [README.md](https://github.com/giantswarm/cert-manager-app/blob/master/README.md) for build instructions
```

Comment:
We usually don't have building instructions in App readme files. IMO we should include some basic instructions